.github/actions/setup-server/action.yml

```diff
@@ -56,6 +60,12 @@ runs:
       shell: bash
       run: python3 scripts/install_local.py -g -s -e
 
+    - name: Update Pypi package version
+      shell: bash
+      run: |
+        echo pypi version (latest release if not specified): ${{ inputs.python-pkg-version }}
+        if [ "${{ inputs.python-pkg-version }}" ]; then echo "${{ inputs.python-pkg-version }} >> $HOME/.aqueduct/server/config/config.yml; fi
```

Comment:
nit: I think this is also achievable with 
```
  - name: Conditional Step
    run: echo "This step runs only if python-pkg-version is set"
    if: ${{ inputs.python-pkg-version }}
```
Which might be a little cleaner. Not the biggest deal.